scripts/install-bluez-5

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+# set -x
+
+pushd /tmp/
```

Comment:
@pgroudas Just FYI, a lot of scripts will drop source in `/usr/local/src` (I used that convention here: https://github.com/garnold/raspberry-pi-recipes/blob/master/scripts/configure-noip-client).  Not sure if that's a standard though.